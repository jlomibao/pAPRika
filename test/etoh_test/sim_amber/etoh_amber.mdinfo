
 NSTEP =    10000   TIME(PS) =      20.000  TEMP(K) =   296.71  PRESS =   340.5
 Etot   =      -390.3008  EKtot   =      3169.7793  EPtot      =     -3560.0801
 BOND   =       303.1423  ANGLE   =      1755.2958  DIHED      =       801.5377
 1-4 NB =       116.9821  1-4 EEL =     -4695.8315  VDWAALS    =     -1855.0278
 EELEC  =        13.8214  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       432.1807  VIRIAL  =        68.6215  VOLUME     =     49445.6122
                                                    Density    =         0.7921
 Ewald error estimate:   0.1568E-01
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

| Final Performance Info:
| -----------------------------------------------------
| Average timings for last     500 steps:
|     Elapsed(s) =      30.92 Per Step(ms) =      61.84
|         ns/day =       2.79   seconds/ns =   30921.37
|
| Average timings for all steps:
|     Elapsed(s) =     613.53 Per Step(ms) =      61.35
|         ns/day =       2.82   seconds/ns =   30676.48
| -----------------------------------------------------

