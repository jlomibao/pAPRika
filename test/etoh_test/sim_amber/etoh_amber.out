
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 02/11/2019 at 13:42:12

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/etoh_test/sim_amber
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: etoh_amber.in                                                         
| MDOUT: etoh_amber.out                                                        
|INPCRD: etoh.rst7                                                             
|  PARM: etoh.prmtop                                                           
|RESTRT: etoh_amber.rst7                                                       
|  REFC: etoh.rst7                                                             
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: etoh_amber.mden                                                       
| MDCRD: etoh_amber.nc                                                         
|MDINFO: etoh_amber.mdinfo                                                     
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          0,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        0,                                                           
  ncyc =          0,                                                           
  dt =            0.002,                                                       
  nstlim =        10000,                                                       
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           9.0,                                                         
  igb =           0,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           1,                                                           
  barostat =      2,                                                           
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  iwrap =         1,                                                           
  ntb =           2,                                                           
  ntwprt =        0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = no_rest.in                                                            
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
|  INFO: Old style inpcrd file read

| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    18.045
| New format PARM file being parsed.
| Version =    1.000 Date = 07/03/18 Time = 16:24:29
 NATOM  =    4608 NTYPES =       5 NBONH =    3072 MBONA  =    1024
 NTHETH =    6144 MTHETA =     512 NPHIH =    8192 MPHIA  =       0
 NHPARM =       0 NPARM  =       0 NNB   =   17408 NRES   =     512
 NBONA  =    1024 NTHETA =     512 NPHIA =       0 NUMBND =       5
 NUMANG =       7 NPTRA  =       5 NATYP =       5 NPHB   =       0
 IFBOX  =       1 NMXRS  =       9 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real              388290
|     Hollerith          14338
|     Integer           549990
|     Max Pairs        2044416
|     nblistReal         55296
|     nblist Int        229063
|       Total            14550 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: RECTILINEAR
Note: ig = -1. Setting random seed to   824870 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       0, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       1, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       2, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   9.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Molecular dynamics:
     nstlim  =     10000, nscm    =      1000, nrespa  =         1
     t       =   0.00000, dt      =   0.00200, vlimit  =  20.00000

Langevin dynamics temperature regulation:
     ig      =  824870
     temp0   = 298.15000, tempi   = 298.15000, gamma_ln=   1.00000

Pressure regulation:
     ntp     =       1
     pres0   =   1.00000, comp    =  44.60000, taup    =   1.00000
     Monte-Carlo Barostat:
     mcbarint  =     100

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       1
     Box X =   37.486   Box Y =   36.090   Box Z =   37.140
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   40       NFFT2 =   36       NFFT3 =   40
     Cutoff=    9.000   Tol   =0.100E-04
     Ewald Coefficient =  0.30768
     Interpolation order =    4

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

TITLE : Created by VMD with 4608 atoms                                          
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = no_rest.in
  LISTOUT   = POUT
 Restraints will be read from file: no_rest.in
Here are comments from the DISANG input file:
                          ** No restraint defined **

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =   0.00051312
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

|  # of SOLUTE  degrees of freedom (RNDFP):   10752.
|  # of SOLVENT degrees of freedom (RNDFS):       0.
|  NDFMIN =   10752.     NUM_NOSHAKE =      0     CORRECTED RNDFP =   10752.
|  TOTAL # of degrees of freedom (RNDF) =   10752.
 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8314E-11   at   2.736960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1161046
| TOTAL SIZE OF NONBOND LIST =    1161046

 NSTEP =        0   TIME(PS) =       0.000  TEMP(K) =   378.39  PRESS =   172.8
 Etot   =       453.0509  EKtot   =      4042.3506  EPtot      =     -3589.2997
 BOND   =       314.5272  ANGLE   =      1743.4368  DIHED      =       789.1969
 1-4 NB =       118.8511  1-4 EEL =     -4668.8769  VDWAALS    =     -1807.3660
 EELEC  =       -79.0687  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       457.4688  VIRIAL  =       270.0463  VOLUME     =     50244.2024
                                                    Density    =         0.7795
 Ewald error estimate:   0.2681E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =      500   TIME(PS) =       1.000  TEMP(K) =   291.90  PRESS =   -47.5
 Etot   =      -446.0461  EKtot   =      3118.3851  EPtot      =     -3564.4312
 BOND   =       316.6099  ANGLE   =      1708.2123  DIHED      =       803.7303
 1-4 NB =       115.0695  1-4 EEL =     -4696.0059  VDWAALS    =     -1843.4964
 EELEC  =        31.4491  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       450.8705  VIRIAL  =       502.4208  VOLUME     =     50306.1951
                                                    Density    =         0.7786
 Ewald error estimate:   0.1406E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     1000   TIME(PS) =       2.000  TEMP(K) =   295.64  PRESS =  -159.9
 Etot   =      -294.7556  EKtot   =      3158.3340  EPtot      =     -3453.0896
 BOND   =       314.4704  ANGLE   =      1787.9884  DIHED      =       797.5225
 1-4 NB =       122.1918  1-4 EEL =     -4676.2260  VDWAALS    =     -1842.0039
 EELEC  =        42.9673  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       465.9741  VIRIAL  =       641.5881  VOLUME     =     50876.8227
                                                    Density    =         0.7698
 Ewald error estimate:   0.2664E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     1500   TIME(PS) =       3.000  TEMP(K) =   294.97  PRESS =  -328.9
 Etot   =      -328.9929  EKtot   =      3151.2067  EPtot      =     -3480.1996
 BOND   =       318.1472  ANGLE   =      1713.0816  DIHED      =       836.8116
 1-4 NB =       127.8167  1-4 EEL =     -4692.2287  VDWAALS    =     -1818.2525
 EELEC  =        34.4245  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       450.2396  VIRIAL  =       812.7447  VOLUME     =     51041.1302
                                                    Density    =         0.7674
 Ewald error estimate:   0.2992E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     2000   TIME(PS) =       4.000  TEMP(K) =   298.33  PRESS =  -192.8
 Etot   =      -323.1666  EKtot   =      3187.0648  EPtot      =     -3510.2313
 BOND   =       311.8077  ANGLE   =      1725.7985  DIHED      =       792.0952
 1-4 NB =       117.0952  1-4 EEL =     -4682.6158  VDWAALS    =     -1771.1708
 EELEC  =        -3.2412  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       487.6771  VIRIAL  =       702.2236  VOLUME     =     51545.7474
                                                    Density    =         0.7599
 Ewald error estimate:   0.1602E+00
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     2500   TIME(PS) =       5.000  TEMP(K) =   300.37  PRESS =  -299.1
 Etot   =      -313.7299  EKtot   =      3208.9484  EPtot      =     -3522.6783
 BOND   =       324.0977  ANGLE   =      1747.8761  DIHED      =       784.1416
 1-4 NB =       111.4258  1-4 EEL =     -4672.5015  VDWAALS    =     -1802.1887
 EELEC  =       -15.5293  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       459.5382  VIRIAL  =       790.8842  VOLUME     =     51302.3982
                                                    Density    =         0.7635
 Ewald error estimate:   0.1419E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     3000   TIME(PS) =       6.000  TEMP(K) =   304.29  PRESS =  -362.1
 Etot   =      -280.0563  EKtot   =      3250.7674  EPtot      =     -3530.8237
 BOND   =       296.7487  ANGLE   =      1749.4930  DIHED      =       795.6753
 1-4 NB =       111.7714  1-4 EEL =     -4677.1732  VDWAALS    =     -1846.6521
 EELEC  =        39.3132  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       476.9954  VIRIAL  =       876.6767  VOLUME     =     51119.5542
                                                    Density    =         0.7662
 Ewald error estimate:   0.3284E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     3500   TIME(PS) =       7.000  TEMP(K) =   297.45  PRESS =   212.3
 Etot   =      -320.5095  EKtot   =      3177.7481  EPtot      =     -3498.2576
 BOND   =       307.0458  ANGLE   =      1790.4370  DIHED      =       801.4846
 1-4 NB =       119.5125  1-4 EEL =     -4679.3104  VDWAALS    =     -1782.8867
 EELEC  =       -54.5404  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       483.4269  VIRIAL  =       251.7889  VOLUME     =     50525.5719
                                                    Density    =         0.7752
 Ewald error estimate:   0.6346E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     4000   TIME(PS) =       8.000  TEMP(K) =   293.50  PRESS =    -4.7
 Etot   =      -465.7380  EKtot   =      3135.5251  EPtot      =     -3601.2631
 BOND   =       311.6544  ANGLE   =      1710.9124  DIHED      =       807.9342
 1-4 NB =       124.5264  1-4 EEL =     -4716.0621  VDWAALS    =     -1808.2233
 EELEC  =       -32.0051  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       449.3646  VIRIAL  =       454.4893  VOLUME     =     50310.3527
                                                    Density    =         0.7785
 Ewald error estimate:   0.2376E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     4500   TIME(PS) =       9.000  TEMP(K) =   299.97  PRESS =  -302.1
 Etot   =      -446.6602  EKtot   =      3204.6282  EPtot      =     -3651.2884
 BOND   =       299.2534  ANGLE   =      1715.1816  DIHED      =       788.7226
 1-4 NB =       118.9086  1-4 EEL =     -4680.1293  VDWAALS    =     -1826.9670
 EELEC  =       -66.2583  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       449.2357  VIRIAL  =       778.2675  VOLUME     =     50438.5681
                                                    Density    =         0.7765
 Ewald error estimate:   0.6924E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     5000   TIME(PS) =      10.000  TEMP(K) =   293.60  PRESS =  -190.1
 Etot   =      -447.3912  EKtot   =      3136.5584  EPtot      =     -3583.9496
 BOND   =       333.7198  ANGLE   =      1730.3636  DIHED      =       780.5819
 1-4 NB =       114.1722  1-4 EEL =     -4696.1370  VDWAALS    =     -1828.6524
 EELEC  =       -17.9978  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       431.3850  VIRIAL  =       638.0313  VOLUME     =     50349.4483
                                                    Density    =         0.7779
 Ewald error estimate:   0.2524E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     5500   TIME(PS) =      11.000  TEMP(K) =   289.43  PRESS =   -62.9
 Etot   =      -446.0744  EKtot   =      3092.0417  EPtot      =     -3538.1161
 BOND   =       361.2374  ANGLE   =      1770.6464  DIHED      =       801.0749
 1-4 NB =       128.1507  1-4 EEL =     -4700.8505  VDWAALS    =     -1817.3625
 EELEC  =       -81.0125  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       407.4000  VIRIAL  =       475.5705  VOLUME     =     50189.4005
                                                    Density    =         0.7804
 Ewald error estimate:   0.1420E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     6000   TIME(PS) =      12.000  TEMP(K) =   303.08  PRESS =   190.6
 Etot   =      -371.5866  EKtot   =      3237.8259  EPtot      =     -3609.4125
 BOND   =       302.7487  ANGLE   =      1753.2531  DIHED      =       779.5310
 1-4 NB =       109.2887  1-4 EEL =     -4668.7659  VDWAALS    =     -1809.2946
 EELEC  =       -76.1734  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       494.5514  VIRIAL  =       287.5142  VOLUME     =     50301.0449
                                                    Density    =         0.7787
 Ewald error estimate:   0.2014E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     6500   TIME(PS) =      13.000  TEMP(K) =   298.81  PRESS =   189.7
 Etot   =      -411.2241  EKtot   =      3192.2829  EPtot      =     -3603.5071
 BOND   =       284.7336  ANGLE   =      1760.2660  DIHED      =       794.1835
 1-4 NB =       123.2468  1-4 EEL =     -4685.8241  VDWAALS    =     -1813.5701
 EELEC  =       -66.5427  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       467.9070  VIRIAL  =       262.7417  VOLUME     =     50091.0137
                                                    Density    =         0.7819
 Ewald error estimate:   0.1026E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     7000   TIME(PS) =      14.000  TEMP(K) =   297.50  PRESS =   151.0
 Etot   =      -461.9933  EKtot   =      3178.2339  EPtot      =     -3640.2272
 BOND   =       297.1965  ANGLE   =      1736.6542  DIHED      =       800.4427
 1-4 NB =       128.5463  1-4 EEL =     -4689.1255  VDWAALS    =     -1809.4658
 EELEC  =      -104.4756  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       449.2154  VIRIAL  =       286.3821  VOLUME     =     49956.7343
                                                    Density    =         0.7840
 Ewald error estimate:   0.3271E-03
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     7500   TIME(PS) =      15.000  TEMP(K) =   304.84  PRESS =   102.8
 Etot   =      -371.9087  EKtot   =      3256.6555  EPtot      =     -3628.5642
 BOND   =       310.5918  ANGLE   =      1779.7137  DIHED      =       802.9729
 1-4 NB =       121.4848  1-4 EEL =     -4677.9653  VDWAALS    =     -1832.5207
 EELEC  =      -132.8415  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       476.9856  VIRIAL  =       366.5452  VOLUME     =     49756.7391
                                                    Density    =         0.7872
 Ewald error estimate:   0.2461E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     8000   TIME(PS) =      16.000  TEMP(K) =   295.08  PRESS =   313.9
 Etot   =      -498.4479  EKtot   =      3152.3451  EPtot      =     -3650.7930
 BOND   =       322.8181  ANGLE   =      1729.8055  DIHED      =       810.0125
 1-4 NB =       121.4522  1-4 EEL =     -4694.8891  VDWAALS    =     -1822.6664
 EELEC  =      -117.3259  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       439.5647  VIRIAL  =       103.7736  VOLUME     =     49539.7347
                                                    Density    =         0.7906
 Ewald error estimate:   0.4465E-03
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     8500   TIME(PS) =      17.000  TEMP(K) =   305.16  PRESS =   137.4
 Etot   =      -336.1269  EKtot   =      3260.1116  EPtot      =     -3596.2386
 BOND   =       298.5708  ANGLE   =      1769.5814  DIHED      =       800.8970
 1-4 NB =       122.7494  1-4 EEL =     -4687.5038  VDWAALS    =     -1836.1419
 EELEC  =       -64.3915  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       490.8080  VIRIAL  =       343.0797  VOLUME     =     49808.9017
                                                    Density    =         0.7864
 Ewald error estimate:   0.6549E-02
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     9000   TIME(PS) =      18.000  TEMP(K) =   296.98  PRESS =    23.0
 Etot   =      -423.7748  EKtot   =      3172.7272  EPtot      =     -3596.5020
 BOND   =       311.5656  ANGLE   =      1773.7500  DIHED      =       785.1372
 1-4 NB =       120.5860  1-4 EEL =     -4688.7327  VDWAALS    =     -1826.5054
 EELEC  =       -72.3027  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       424.2117  VIRIAL  =       399.3120  VOLUME     =     50142.9342
                                                    Density    =         0.7811
 Ewald error estimate:   0.1309E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =     9500   TIME(PS) =      19.000  TEMP(K) =   297.06  PRESS =     1.9
 Etot   =      -461.0435  EKtot   =      3173.5623  EPtot      =     -3634.6059
 BOND   =       307.8653  ANGLE   =      1727.3266  DIHED      =       810.2977
 1-4 NB =       119.3914  1-4 EEL =     -4685.6131  VDWAALS    =     -1852.1637
 EELEC  =       -61.7101  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       458.8962  VIRIAL  =       456.8450  VOLUME     =     49774.4389
                                                    Density    =         0.7869
 Ewald error estimate:   0.1629E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

 NSTEP =    10000   TIME(PS) =      20.000  TEMP(K) =   296.71  PRESS =   340.5
 Etot   =      -390.3008  EKtot   =      3169.7793  EPtot      =     -3560.0801
 BOND   =       303.1423  ANGLE   =      1755.2958  DIHED      =       801.5377
 1-4 NB =       116.9821  1-4 EEL =     -4695.8315  VDWAALS    =     -1855.0278
 EELEC  =        13.8214  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       432.1807  VIRIAL  =        68.6215  VOLUME     =     49445.6122
                                                    Density    =         0.7921
 Ewald error estimate:   0.1568E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

      A V E R A G E S   O V E R   10000 S T E P S


 NSTEP =    10000   TIME(PS) =      20.000  TEMP(K) =   297.62  PRESS =   -54.8
 Etot   =      -392.4762  EKtot   =      3179.5597  EPtot      =     -3572.0359
 BOND   =       311.9524  ANGLE   =      1747.9657  DIHED      =       796.2201
 1-4 NB =       119.0664  1-4 EEL =     -4685.6098  VDWAALS    =     -1821.4680
 EELEC  =       -40.1627  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =       454.0715  VIRIAL  =       514.1007  VOLUME     =     50382.4808
                                                    Density    =         0.7775
 Ewald error estimate:   0.4159E-01
 ------------------------------------------------------------------------------

 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================

      R M S  F L U C T U A T I O N S


 NSTEP =    10000   TIME(PS) =      20.000  TEMP(K) =     3.75  PRESS =   225.7
 Etot   =        65.6584  EKtot   =        40.0971  EPtot      =        59.7598
 BOND   =        13.7400  ANGLE   =        31.1194  DIHED      =        11.3269
 1-4 NB =         4.1533  1-4 EEL =        10.8108  VDWAALS    =        30.1779
 EELEC  =        50.9620  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 EKCMT  =        18.2851  VIRIAL  =       243.1510  VOLUME     =       501.3991
                                                    Density    =         0.0077
 Ewald error estimate:   0.5193E+00
 ------------------------------------------------------------------------------


 NMR restraints on final step:

 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: etoh_amber.rst7                         


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
|                               RMS deviation from ideal bonds :      0.0156
|                               RMS deviation from ideal angles:      4.356
 ------------------------------------------------------------------------------
| MC Barostat:        100 volume changes attempted.
| MC Barostat:         55 changes successful ( 55.00%)
 ------------------------------------------------------------------------------


--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Build the list            34.44 (99.21% of List )
|                Other                      0.28 ( 0.79% of List )
|             List time                 34.72 ( 6.12% of Nonbo)
|                   Short_ene time           450.77 (99.35% of Direc)
|                   Other                      2.95 ( 0.65% of Direc)
|                Direct Ewald time        453.73 (85.18% of Ewald)
|                Adjust Ewald time         10.03 ( 1.88% of Ewald)
|                   Fill Bspline coeffs        3.31 ( 5.03% of Recip)
|                   Fill charge grid           7.69 (11.71% of Recip)
|                   Scalar sum                15.03 (22.87% of Recip)
|                   Grad sum                  16.36 (24.90% of Recip)
|                   FFT time                  23.29 (35.45% of Recip)
|                   Other                      0.02 ( 0.04% of Recip)
|                Recip Ewald time          65.70 (12.33% of Ewald)
|                Force Adjust               0.07 ( 0.01% of Ewald)
|                Virial junk                0.21 ( 0.04% of Ewald)
|                Other                      2.89 ( 0.54% of Ewald)
|             Ewald time               532.64 (93.88% of Nonbo)
|          Nonbond force            567.37 (95.08% of Force)
|          Bond/Angle/Dihedral       29.18 ( 4.89% of Force)
|          Other                      0.19 ( 0.03% of Force)
|       Force time               596.74 (97.26% of Runmd)
|       Shake time                 2.60 ( 0.42% of Runmd)
|       Verlet update time        13.67 ( 2.23% of Runmd)
|       Ekcmr time                 0.25 ( 0.04% of Runmd)
|       Other                      0.27 ( 0.04% of Runmd)
|    Runmd Time               613.53 (100.0% of Total)
|    Other                      0.05 ( 0.01% of Total)
| Total time               613.58 (100.0% of ALL  )

| Number of list builds   :        798

| Highest rstack allocated:     256494
| Highest istack allocated:       4608

| Final Performance Info:
| -----------------------------------------------------
| Average timings for all steps:
|     Elapsed(s) =     613.53 Per Step(ms) =      61.35
|         ns/day =       2.82   seconds/ns =   30676.50
| -----------------------------------------------------

|           Job began  at 13:42:12.780  on 02/11/2019
|           Setup done at 13:42:12.832  on 02/11/2019
|           Run   done at 13:52:26.362  on 02/11/2019
|     wallclock() was called  469267 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
